nil/client/mock_Client.go

```diff
@@ -0,0 +1,2436 @@
+// Code generated by mockery v2.53.3. DO NOT EDIT.
```

Comment:
We don't commit autogenerated files into repository. Please add them to gitignore.